express-notes-api/index.js

```diff
@@ -0,0 +1,146 @@
+const express = require('express');
+const fs = require('fs');
+const data = require('./data.json');
+const app = express();
+
+app.get('/api/notes', (req, res) => {
+  const notes = [];
+  for (const id in data.notes) {
+    const note = data.notes[id];
+    notes.push(note);
+  }
+  if (notes.length === 0) {
+    return [];
+  } else {
+    res.status(200).json(notes);
+  }
```

Comment:
You can respond with the notes either way, if the array length is `0` that's fine.